src/react_native/gesture.cljs

```diff
@@ -1,6 +1,8 @@
 (ns react-native.gesture
-  (:require ["react-native-gesture-handler" :refer (GestureDetector Gesture gestureHandlerRootHOC)]
-            [reagent.core :as reagent]))
+  (:require ["react-native-gesture-handler" :refer
+             (GestureDetector Gesture gestureHandlerRootHOC TapGestureHandler State)]
```

Comment:
But that would make us import the entire library, right? Shouldn't refer be better here?

Reply:
I'm not completely sure of the answer because I haven't looked at the compiled & optimized JS generated by shadow-cljs. But I just did generate a report by running `npx shadow-cljs run shadow.cljs.build-report mobile report.html` and the output didn't include a single byte coming from ReactNative libraries. Maybe the command is meaningless for RN, but I was surprised tbh.

In web development this concern is quite important, and to avoid increasing the bundle size too much we need to require submodules of a library. Using only `refer` is not enough, so I suspect it makes no different using the alias or not here in a ReactNative codebase. The Google Closure compiler with advanced optimizations enabled will also remove code that is never executed (or try to...), and our `mobile` build has advanced optimizations enabled, see https://github.com/status-im/status-mobile/blob/develop/shadow-cljs.edn#L73

Reply:
@ilmotta so is it okay to use `refer` here? I see most of the files in the `react-native` module using refer. 
Using `as` creates unnecessary repeated duplications of `gesture-handler/..`, wdyt?

Reply:
After reviewing the rest of the files in the react-native directory, I see whenever multiple imports are needed from a module `refer` is used (except in one file svg.cljs). For consistency sake I will keep it is as refer for now. 

The rest of the imports can be fixed in a separate PR to use one style or the other (probably after hearing from others also). I don't mind any style, but `refering` RN imports seems to be a bit cleaner.

Reply:
Totally agree Omar, not something for your PR. But I thought this discussion had already reached consensus. It doesn't matter if it's a react native library or not, we aligned that namespaces should be aliased and `refer` should only be used for widely known aliases, like in test namespaces.